elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -609,6 +610,22 @@ elytron.simple-role-decoder.remove=Remove operation for the RoleDecoder
 # Attributes
 elytron.simple-role-decoder.attribute=The name of the attribute from the identity to map directly to roles.
 
+elytron.source-address-role-decoder=Definition of a RoleDecoder that maps roles based on the IP address of a remote client.
+# Operations
+elytron.source-address-role-decoder.add=Add operation for the RoleDecoder
+elytron.source-address-role-decoder.remove=Remove operation for the RoleDecoder
+# Attributes
+elytron.source-address-role-decoder.source-address=The IP address to match. Exactly one of 'address' and 'pattern' must be specified.
```

Comment:
I'm not sure you need to specify "Exactly one of 'address' and 'pattern' must be specified". Their metadata states that they are alternatives so that's enough.
Having said that, I also have a doubt whether they should be flagged as required too....

Reply:
Removed that sentence.